### DISCUSSION TITLE
umi2升级到umi3时报错 set.clone is not a function

### DISCUSSION THREAD

**Discussion by longxin-vico** [umijs/umi#10723] [Q&A]
## 背景
项目从umi2升级到umi3，按照官方升级指南更改之后，剩最后一个报错没有找到方法。
<img width="1051" alt="image" src="https://user-images.githubusercontent.com/8231744/224256829-0846c45c-bb2b-45a0-a3f9-5b577867ee48.png">

调试发现set是/node_modules/@umijs/deps/compiled/babel/bundle.js里面的getUnicodePropertyEscapeSet函数赋值, set.clone = undefined导致报错，根本原因是什么导致set.clone = undefined。
<img width="642" alt="image" src="https://user-images.githubusercontent.com/8231744/224257363-540f07f9-2841-45f8-ab1f-89787e0695a3.png">

**Comment by fz6m** [comment]
排查下失败的这个页面，里面的写法可能有问题：•/src/pages/bond-sales/utils/ index.ts
